tools/scripts/Makefile

```diff
@@ -29,25 +42,38 @@ MUTE = @
 NULL = >/dev/null
 endif
 
+#------------------------------------------------------------------------------
+#                                ECHO HELPER                                   
+#------------------------------------------------------------------------------
 EXEC = release
 TIMESTAMP = $(shell date +"%T")
 define print
-	@printf "\\e[32m[$(TIMESTAMP)]\\e[39m $(1)"
+	printf "\\e[32m[$(TIMESTAMP)]\\e[39m $(1)"
 endef
 define print_err
-	@printf "\\e[32m[$(TIMESTAMP)]\\e[31m $(1)"
+	printf "\\e[32m[$(TIMESTAMP)]\\e[31m $(1)"
 endef
+
+#------------------------------------------------------------------------------
+#                              DEFAULT COMPILER                                
+#------------------------------------------------------------------------------
 CC ?= gcc
 
-# Xilinx platform handling
-ifeq (xilinx,$(strip $(PLATFORM)))
+#------------------------------------------------------------------------------
+#                              DEFAULT LINKER                                  
+#------------------------------------------------------------------------------
+LD ?= $(CC)
 
+#------------------------------------------------------------------------------
+#                            COMMON COMPILER FLAGS                             
+#------------------------------------------------------------------------------
 CFLAGS = -Wall 								\
 	 -Wmissing-field-initializers					\
 	 -Wclobbered 							\
 	 -Wempty-body 							\
 	 -Wignored-qualifiers 						\
 	 -Wmissing-parameter-type					\
+	 -Wno-format  							\
```

Comment:
not related to this commit

Reply:
I still see unrelated changes...

Reply:
This flag is used by default in the intel sdk